cli/internal/lockfile/npm_lockfile.go

```diff
@@ -44,6 +44,9 @@ type NpmPackage struct {
 	Dependencies         map[string]string `json:"dependencies,omitempty"`
 	DevDependencies      map[string]string `json:"devDependencies,omitempty"`
 	PeerDependencies     map[string]string `json:"peerDependencies,omitempty"`
+	PeerDependenciesMeta map[string]struct {
+		Optional bool `json:"optional"`
```

Comment:
I think you want to only want entries if `optional` is set to `true`. (Probably unimportant as npm is fairly relaxed with the exact representation in the lockfile)
```suggestion
		Optional bool `json:"optional,omitempty"`
```

Reply:
I wondered about that, thanks for the suggestion!